src/components/FlashMessage/FlashMessage.stories.tsx

```diff
@@ -135,7 +133,7 @@ export const Demo: StoryFn = () => {
       <hr />
       <fieldset>
         <legend>animation</legend>
-        {animationTypes.map((animationType) => (
+        {['bounde', 'none'].map((animationType) => (
```

Comment:
```suggestion
        {['bounce', 'none'].map((animationType) => (
```

おそらくtypoかなと！ ([Storybook](https://63d0ccabb5d2dd29825524ab-dwyqhppxua.chromatic.com/?path=/story/states%EF%BC%88%E7%8A%B6%E6%85%8B%EF%BC%89-flashmessage%EF%BC%88%E9%9D%9E%E6%8E%A8%E5%A5%A8%EF%BC%89--demo)上でも動いてなさそう)

Reply:
ありがとうございます! 修正しました! [d70c7ec](https://github.com/kufu/smarthr-ui/pull/4547/commits/d70c7ec45d986ae9e19569ca37945d1fc23319d9)